compositions/NFTPage/NFTPage.css.ts

```diff
@@ -194,3 +194,20 @@ export const nftAttribute = style([
     },
   }),
 ])
+
+export const nftNextButton = style([
+  {
+    borderRadius: '50%',
+    padding: 0,
+    marginLeft: '.5rem',
```

Comment:
Many of these can be implemented atoms to reduce the CSS footprint.

eg 
borderRadius: round
padding: 'x0'
display
fontSize

Is the spacing based on the zord spacing? If so, marginLeft, width, and height should all be atoms too.

Reply:
addressed, when possible
padding: 'x0' is buggy, discussed here https://ourzora.slack.com/archives/C03KSUKQX0U/p1661902550137249